### PR TITLE
WT-13291 Fix timestamp buffers and logs in RTS

### PR DESCRIPTION
**Pull request by etienneptl**

### CODE REVIEW

File: src/rollback_to_stable/rts_btree_walk.c [hunk 1]
```diff
@@ -104,13 +104,14 @@ __rts_btree_walk_page_skip(
         __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
           WT_RTS_VERB_TAG_STABLE_PG_WALK_SKIP "ref=%p: stable page walk skipped", (void *)ref);
         WT_STAT_CONN_INCR(session, txn_rts_tree_walk_skip_pages);
-    }
+    } else {
+        reconciled = ref->page && ref->page->modify ? true : false;
```
Comment (etienneptl) [inline src/rollback_to_stable/rts_btree_walk.c:108]: Is there any reason why we want to print the `reconciled` property only in this code path?

**Review by quchenhao — APPROVED** [review]
lgtm

File: src/rollback_to_stable/rts_btree_walk.c (hunk 1 above)
Comment (kommiharibabu) [reply src/rollback_to_stable/rts_btree_walk.c:108]: in this flow, the page is on disk, we shouldn't have a `ref->page` itself.

Comment (etienneptl) [reply src/rollback_to_stable/rts_btree_walk.c]: Removed in aeac5f015904e4bdd5ed7719deec3d004fc2c37e

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Review by mm-ng — APPROVED** [review]
LGTM

**Pull request merged**